src/main/java/uk/antiperson/stackmob/StackMob.java

```diff
@@ -141,7 +147,7 @@ private void registerEvents() throws InvocationTargetException, NoSuchMethodExce
         registerEvent(TargetListener.class);
         registerEvent(PlayerListener.class);
         registerEvent(BeeListener.class);
-        registerEvent(LeashListener.class);
```

Comment:
Why did you remove this?

Reply:
It was accidental, I added it back in a later commit 6ca24828707b304c167c78fe4fb8d17739a76f61